src/components/common/Typography/TextArea/TextArea.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TextArea = ({ className, value, onChange }) => (
```

Comment:
This will share all the props to the typography component.
```suggestion
const TextArea = (props) => (
  <textarea {...props} />
);
```